fem/integ/bilininteg_elasticity_pa.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "../bilininteg.hpp"
+#include "../gridfunc.hpp"
+#include "../qfunction.hpp"
+#include "bilininteg_elasticity_kernels.hpp"
+
+namespace mfem
+{
+
+
+void ElasticityIntegrator::AssemblePA(const FiniteElementSpace &fes)
+{
+   MFEM_VERIFY(fes.GetOrdering() == Ordering::byNODES,
+               "Elasticity PA only implemented for byNODES ordering.");
+   if (!parent)
+   {
+      fespace = &fes;
+   }
+   else
+   {
+      MFEM_VERIFY(parent->PACalled,
+                  "Parent integrator needs to have been partially assembled.");
+   }
+
+   if (!parent)
+   {
+      const auto el = fes.GetFE(0);
+      ndofs = el->GetDof();
+      const auto mesh = fes.GetMesh();
+      vdim = fes.GetVDim();
+      const IntegrationRule *ir = IntRule;
+      if (ir == NULL)
+      {
+         //This is where it's assumed that all elements are the same.
+         const auto Trans = fes.GetElementTransformation(0);
+         int order = 2 * Trans->OrderGrad(el);
+         IntRule = &IntRules.Get(el->GetGeomType(), order);
+      }
+      geom = mesh->GetGeometricFactors(*IntRule, GeometricFactors::JACOBIANS);
+      quad_space = std::make_shared<QuadratureSpace>(*mesh, *IntRule);
+      lambda_quad = std::make_shared<QuadratureFunction>(*quad_space);
+      mu_quad = std::make_shared<QuadratureFunction>(*quad_space);
+      q_vec = std::make_shared<QuadratureFunction>(*quad_space, vdim*vdim);
+      lambda->Project(*lambda_quad);
+      mu->Project(*mu_quad);
+      auto ordering = UsesTensorBasis(*fespace) ? ElementDofOrdering::LEXICOGRAPHIC :
+                      ElementDofOrdering::NATIVE;
```

Comment:
We may want this to become a function since it's not really a user choice, .e.g. `GetEVectorOrdering()`, but this does not necessarily needs to happen in this PR.

Reply:
I may no longer actually need this if the `LEXICOGRAPHIC_FULL` idea works so hold off on reviewing this part.